doc/api/child_process.markdown

```diff
@@ -483,7 +499,7 @@ spawn('prg', [], { stdio: ['pipe', null, null, null, 'pipe'] });
 *It is worth noting that when an IPC channel is established between the
 parent and child processes, and the child is a Node.js process, the child
 is launched with the IPC channel unreferenced (using `unref()`) until the
-child registers an event handler for the `process.on('disconnected')` event.
+child registers an event handler for the [`process.on('disconnect')`][] event.
```

Comment:
Yes, a search of the codebase turns up `emit('disconnect')`, but not `disconnected`.